packages/webapp/src/components/landing-page/LandingPage.tsx

```diff
@@ -116,6 +117,16 @@ const LandingPage = (props: TLandingPageProps) => {
         if (boardId) setBoardId(boardId);
     }, [location.search, setBoardId]);
 
+    function Copyright() {
+        return (
+            <Typography variant="body2" color="textSecondary" align="center">
+            {'From the minds of '}
+            <Link color="primary" href="https://github.com/orgs/bored-engineers/people">
```

Comment:
Link will open the page in same tab, we can use anchor tag instead. 
ref [see here](https://github.com/bored-engineers/the-retro-app/blob/master/packages/webapp/src/components/common/footer/Footer.tsx#L14)